hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/input/TestLineRecordReaderBZip2.java

```diff
@@ -22,8 +22,13 @@
 public final class TestLineRecordReaderBZip2
     extends BaseTestLineRecordReaderBZip2 {
 
+  @Override
+  public void setUp() throws Exception {
+    super.setUp();
+  }
```

Comment:
I think the lines are not required for the checkstyle fix.

Reply:
Thanks @aajisaka - I have addressed the above comment.